src/Adapter/Product/CommandHandler/UpdateProductBasicInformationHandler.php

```diff
@@ -49,15 +50,24 @@ class UpdateProductBasicInformationHandler implements UpdateProductBasicInformat
      */
     private $defaultLanguageId;
 
+    /**
+     * @var Tools
+     */
+    private $tools;
+
     /**
      * @param ProductMultiShopRepository $productRepository
+     * @param int $defaultLanguageId
+     * @param Tools $tools
      */
     public function __construct(
         ProductMultiShopRepository $productRepository,
-        int $defaultLanguageId
+        int $defaultLanguageId,
+        Tools $tools
```

Comment:
Please define the BC Break as the class exists on `1.7.8.x` and is not defined as `@internal`.